io_scene_dae/export_dae.py

```diff
@@ -294,103 +294,104 @@ def export_material(self, material, double_sided_hint=True):
 
-        self.writel(S_FX, 5, '<shininess>')
-        self.writel(S_FX, 6, '<float>' + str(material.specular_hardness) +
-                    '</float>')
-        self.writel(S_FX, 5, '</shininess>')
+        self.writel(S_FX, 5, "<shininess>")
+        self.writel(S_FX, 6, "<float>{}</float>".format(
+            material.specular_hardness))
+        self.writel(S_FX, 5, "</shininess>")
 
-        self.writel(S_FX, 5, '<reflective>')
-        self.writel(S_FX, 6, '<color>' + numarr_alpha(material.mirror_color) +
-                    '</color>')
-        self.writel(S_FX, 5, '</reflective>')
+        self.writel(S_FX, 5, "<reflective>")
+        self.writel(S_FX, 6, "<color>{}</color>".format(
+            numarr_alpha(material.mirror_color)))
+        self.writel(S_FX, 5, "</reflective>")
 
         if (material.use_transparency):
-            self.writel(S_FX, 5, '<transparency>')
-            self.writel(S_FX, 6, '<float>' + str(material.alpha) + '</float>')
-            self.writel(S_FX, 5, '</transparency>')
+            self.writel(S_FX, 5, "<transparency>")
+            self.writel(S_FX, 6, "<float>{}</float>".format(material.alpha))
+            self.writel(S_FX, 5, "</transparency>")
 
-        self.writel(S_FX, 5, '<index_of_refraction>')
-        self.writel(S_FX, 6, '<float>' + str(material.specular_ior) +
-                    '</float>')
-        self.writel(S_FX, 5, '</index_of_refraction>')
+        self.writel(S_FX, 5, "<index_of_refraction>")
+        self.writel(S_FX, 6, "<float>{}</float>".format(material.specular_ior))
+        self.writel(S_FX, 5, "</index_of_refraction>")
 
-        self.writel(S_FX, 4, '</' + shtype + '>')
+        self.writel(S_FX, 4, "</{}>".format(shtype))
 
-        self.writel(S_FX, 4, '<extra>')
-        self.writel(S_FX, 5, '<technique profile="FCOLLADA">')
+        self.writel(S_FX, 4, "<extra>")
+        self.writel(S_FX, 5, "<technique profile=\"FCOLLADA\">")
         if (normal_tex):
-            self.writel(S_FX, 6, '<bump bumptype="NORMALMAP">')
-            self.writel(S_FX, 7, '<texture texture="' + normal_tex +
-                        '" texcoord="CHANNEL1"/>')
-            self.writel(S_FX, 6, '</bump>')
+            self.writel(S_FX, 6, "<bump bumptype=\"NORMALMAP\">")
+            self.writel(
+                S_FX, 7,
+                "<texture texture=\"{}\" texcoord=\"CHANNEL1\"/>".format(
+                    normal_tex))
+            self.writel(S_FX, 6, "</bump>")
 
-        self.writel(S_FX, 5, '</technique>')
-        self.writel(S_FX, 5, '<technique profile="GOOGLEEARTH">')
-        self.writel(S_FX, 6, '<double_sided>' + ["0", "1"][double_sided_hint] +
-                    "</double_sided>")
-        self.writel(S_FX, 5, '</technique>')
+        self.writel(S_FX, 5, "</technique>")
+        self.writel(S_FX, 5, "<technique profile=\"GOOGLEEARTH\">")
+        self.writel(S_FX, 6, "<double_sided>{}</double_sided>".format(
+            ["0", "1"][double_sided_hint]))
```

Comment:
```python
["0", "1"][double_sided_hint]
```

This code is a bit weird; since `double_sided_hint` is a boolean, maybe replacing it with `int(double_sided_hint)` is easier to read and understand (no need to call `str` since we're in `format`)?